### PR TITLE
Update docs with current commands and palettes

### LINKED ISSUE #240: Docs outdated when listing sub-commands
After `status` sub-command was reworked by https://github.com/src-d/sourced-ce/pull/208, and `sourced` [v0.15.0](https://github.com/src-d/sourced-ce/releases/tag/v0.15.0) was released, `workdirs` sub-command no longer exists.

Docs are now outdated, e.g. [usage/commands#sourced-workdirs](https://docs.sourced.tech/community-edition/usage/commands#sourced-workdirs)

### PR DESCRIPTION
**Pull request by dpordomingo**
fix #240

This PR updates:

- command list docs
- screenshot with new source{d} branded palette.

---

* [ ] I have updated the CHANGELOG file according to the conventions in [keepachangelog.com](https://keepachangelog.com)
* [x] This PR contains changes that do not require a mention in the CHANGELOG file

### CODE REVIEW

File: docs/usage/commands.md [hunk 1]
```diff
@@ -115,7 +129,7 @@ You can activate any other with `compose set`.
 
 ### sourced compose set
 
-Sets the active `docker-compose.yml` file.
+Sets the active `docker-compose.yml` file providing either the name or index of the compose file to activate as you can get from 'compose list'.
```
Comment (carlosms) [inline docs/usage/commands.md]:
```suggestion
Sets the active `docker-compose.yml` file. Accepts either the name or index of the compose file as returned by 'compose list'.
```

**Review by carlosms — APPROVED** [review]
Maybe the image can be made smaller? It's huge, but then github and gitbook will render with with a smaller fixed width.

**Comment by dpordomingo** [comment]
The image is the same than before, but with the current color palette. However, I just pushed a smaller 1600px new image, which still lets the user enlarge it in GitBook without losing too much detail. 

example: https://dpordomingo.gitbook.io/community-edition/

Anyhow, we're still waiting for the final image from @src-d/product. This is only our best attempt of having better readme as possible.

**Comment by ricardobaeta** [comment]
> Anyhow, we're still waiting for the final image from @src-d/product. This is only our best attempt of having better readme as possible.

I'm fine with this image, I would just add a soft grey drop-shadow. Do you want it?

**Comment by dpordomingo** [comment]
The only constraint is to fit in both GitBook and GitHub docs.
https://docs.sourced.tech/community-edition
https://github.com/src-d/sourced-ce

If it does, whatever image you produce, will lgtm.
(but I'd prefer not to block this PR till we have a new image if the current one is ok; if the new one needs extra time to be produced, it could be merged in separate PR, just overriding [the current one](https://github.com/src-d/sourced-ce/blob/master/docs/assets/dashboard.png))

**Comment by ricardobaeta** [comment]
> (but I'd prefer not to block this PR till we have a new image if the current one is ok; if the new one needs extra time to be produced, it could be merged in separate PR, just overriding the current one)

By all means @dpordomingo :)

**Comment by dpordomingo** [comment]
merged as confirmed via slack. Will iterate on the screenshot in another PR.

**Pull request merged**